save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/ExtraFlagsExtractor.kt

```diff
@@ -20,10 +21,10 @@ class ExtraFlagsExtractor(private val generalConfig: GeneralConfig,
         val allExtraFlagsFromFile = fs.readLines(path).mapNotNull {
             extractExtraFlagsFrom(it)
         }
-        require(allExtraFlagsFromFile.size <= 1) {
+        logDebug(
```

Comment:
I think it should be logged with higher level, at least Info or maybe even Warn

Reply:
no, unfortunately Warn is too overloaded now

Reply:
I think it will be important both for us and for future users to see why run configurations are not taken into account. Maybe we should create a new issue for improving logging and reconsidering levels for all messages?